### ISSUE TITLE
add metadata fields to the annotation UI

### ISSUE THREAD

**Issue by monkeypants** [Tech4Tracing/CartridgeOCR#98]
* calibre
* country (of manufacture)
* manufacturer
* factory
* year of manufacture
* local reference (private field)
* URL (was this image found online?)
* remarks (where did you get it the headstamp?)

**Comment by simra** [comment]
I realized after the call that we haven't yet landed on a good workflow for distinguishing between an image and the one or more headstamps that might be in it.  For my annotations I actually ran the detector offline and uploaded individual images of each detected headstamp.  I think this works fine for my workflow but for collectors we might need something more nuanced. 

Also, in our discussion I talked a lot about an existing metadata field but this actually applies to annotations- we would need to add a new metadata field for images.  Not a big issue but it will be a schema change in the database.
